istio-control/istio-discovery/templates/telemetryv2.yaml

```diff
@@ -1,5 +1,234 @@
 {{- if and .Values.telemetry.enabled .Values.telemetry.v2.enabled }}
-{{ .Files.Get "metadata-exchange-v2.yaml" }}
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
This should be enabled if either stackdriver or prom is enabled.

Reply:
then there is no meaning of the v2.enabled setting? also if we have some more other filters in the future then this check would become lengthy

Reply:
Ok, that makes sense, this is only enabled when v2 is enabled .